docs/user-guides/contributors_guide.md

```diff
@@ -6,38 +6,57 @@ We love your input! We want to make contributing to this project as easy and tra
 
 
-
 1. [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) and create your branch from `master`.
 2. If you've added code that should be tested, add tests.
 3. Ensure the test suite passes.
 4. Make sure your code lints.
-5. Issue that pull request!
+5. Issue raised related to the same should be mentioned in pull request!
+
+___________________________________________________
+
+## Bugs and New Features
+
+We keep a distinction between New Features and Bugs/Issues. We try to minimise the noise in our Github Issues stream by first having a conversation on approprate media (eg:  [KochiOrgBook! Whatsapp Group](https://chat.whatsapp.com/El6d3aAc6pYLEDEdQPVABY). ) Initial discussion of new features or enhancements can be discussed in this whatsapp channel
 
-## Report bugs using Github's [issues](https://github.com/hyperledgerkochi/KOBman/issues)
+### Report bugs using Github's issues
 We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/hyperledgerkochi/KOBman/issues); it's that easy!
+After confirming the bug was found, an Issue may be raised filling in the provided issue template on the KOBman issue tracker.
 
+### Process to add new feature
 
-### Bugs and New Features
+ - Explain the development requirements with the KOBman community
+ - Start writing automation , by adding the new feature code and it's test script. 
+ - Ensure that the entire community is aware of it to avoid duplication of your effort, by publishing your feature request in [Kanban board](https://github.com/orgs/hyperledgerkochi/projects/3) , [Issue Section](https://github.com/hyperledgerkochi/KOBman/issues)
 
-We keep a distinction between New Features and Bugs/Issues. We try to minimise the noise in our Github Issues stream by first having a conversation on approprate media (eg:  [KochiOrgBook! Whatsapp Group](https://chat.whatsapp.com/El6d3aAc6pYLEDEdQPVABY). ) 
+_____________________________________
 
-- User Issues can be raised in our [issues section](https://github.com/hyperledgerkochi/KOBman/issues).
-- New Features or Enhancements can be discussed in our issue section.
 
-After accepting a new feature or confirming that a bug was found, an Issue may be raised filling in the provided issue template on the KOBman issue tracker.
+## Processes expected to follow by each contributors:
+
+- [Process for pull request](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-a-pull-request.md)
+
+- [Process for issues](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-an-issue.md)
+
+_______________________________________
 
 **Unfortunately we will close any Github Issues that have not followed this process of prior discussion.**
 
 ## Write bug reports with detail, background, and sample code
 [This is an example](https://github.com/asa1997/KOBman/issues/8) of a bug reported by a contributor .

```

Comment:
@EtricKombat Please use the link for the issue section under hyperledgerkochi/KOBman